Makefile

```diff
@@ -125,7 +139,10 @@ check-certs: # Check cert subject
 	openssl x509 -noout -in /tmp/cert.pem --subject
 
 ##@ Build
-build: generate fmt vet ## Build manager binary.
+build: generate fmt vet gosec imports ## Build manager binary.
+	go build -mod=vendor -o bin/manager main.go
```

Comment:
Now that gosec is running, it produces a lot of errors.
It would be simpler to exclude gosec from being invoked as part of the `build` rule, make sure this PR is ready to be merged and after it is merged, continue with enabling `gosec` as a follow-up item.